docs/hazmat/primitives/mac/poly1305.rst

```diff
@@ -0,0 +1,85 @@
+.. hazmat::
+
+Poly1305
+========
+
+.. currentmodule:: cryptography.hazmat.primitives.poly1305
+
+.. testsetup::
+
+    key = b"\x01" * 32
+
+Poly1305 is a one-time authenticator that takes a 32-byte one-time key and a
+message and produces a 16-byte tag.  This tag is used to authenticate the
+message. It is described in :rfc:`7539`.
+
+.. class:: Poly1305(key)
+
+    .. versionadded:: 2.7
+
+    .. warning::
+
+        Reuse of a key potentially allows forgery of authentication tags. Always
```

Comment:
I think this reads more clearly: "Using the same key to sign multiple messages allows an attacker to forge future tags" or something